src/lib/y2users/password.rb

```diff
@@ -222,21 +259,60 @@ def force_change
     #   - password aging features are disabled (see {#enabled?})
     #   - the user is forced to change the password in the next login (see {#force_change?})
     #
+    # @see ShadowDateField#to_date
+    #
     # @return [Date, nil]
     def last_change
       return nil unless enabled?
       return nil if force_change?
 
-      shadow_string_to_date(content)
+      to_date
     end
 
     # Sets the content to the given date
     #
     # @note This enables the password aging features and sets {#force_change?} to false.
     #
+    # @see ShadowDateField#to_shadow
+    #
     # @param date [Date]
     def last_change=(date)
-      self.content = date_to_shadow_string(date)
+      @content = to_shadow(date)
+    end
+  end
+
+  # Represents the account expiration field of the shadow file
+  class AccountExpiration < ShadowDateField
+    # Whether the account has an expiration date
+    #
+    # @return [Boolean]
+    def expire?
+      !content.empty?
+    end
+
+    # Disables the account expiration
```

Comment:
Maybe we should also explain here that there is not an `#enable` counterpart and that `date=` should be used instead